app.js

```diff
@@ -34,6 +38,43 @@ compileSVGSprites(
   'image'
 )
 
+// config express-session
+const theSession = {
+  secret: '72a1e6ba625661ee10dfd9610abb5499acc276da06375e56ed2d3789d0aa11ae',
```

Comment:
this is, i think, a locally generated token so I don't know if it needs to not be committed like other secrets